### DISCUSSION TITLE
Can you set a path for large monorepos / megarepos?

### DISCUSSION THREAD

**Discussion by jugghayd** [zricethezav/gitleaks#1388] [General]
We use GitLeaks for DevSecOps, but some of our repos are huge with 50+ solutions in them. We want to narrow down the GitLeaks to the particular folder in a CI/CD pipeline to make the scan step faster. It looks like we can set a path on each individual rule, but can we set a parent folder on a more global level to only scan inside of?

.e.g.
Megarepo\AppleApp
Megarepo\BananaApp
Megarepo\CherryApp

And I only want to scan CherryApp and its child folders and files

**Comment by rgmz** [comment]
https://github.com/gitleaks/gitleaks/issues/1163#issuecomment-1522019039 may be a relevant solution.
